docs/custom_loading.rst

```diff
@@ -166,6 +169,10 @@ This again will be done in a separate python file and placed in the user's
 
         tab.write(file_name, format="fits")
 
+Please note that it is assumed that the axis (of the ``flux`` and ``spectral_axis``)
+units information is stored in the ``meta`` parameter as introduced in the definition
+of ``generic-fits`` as the ``custom_writer()`` above.
+
```

Comment:
This is not correct. In the general case we should not make any assumptions on the information stored in the `meta` attribute of a `Spectrum1D` object. However the units _are_ guaranteed to be already part of `flux` and `spectral_axis`, because the are verified to be `Quantity`instances on initialisation. That's why just
```
        flux = spectrum.flux
        disp = spectrum.spectral_axis
```
should be used.

Reply:
I have changed the wording to match your descriptions.